src/client/components/dataItem/DataItem.jsx

```diff
@@ -15,17 +15,15 @@ export const DataItem = ({
       <span>
```

Comment:
Just a note for later maybe: this seems like a good component for snapshot testing.